pkg/cidata/cidata.TEMPLATE.d/boot/07-host-dns-setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -eux
+
+# Wait until iptables has been installed; 30-install-packages.sh will call this script again
```

Comment:
Why not just call this script after installing iptables?

Reply:
When `useHostResolver` is true, we can't set the `en0` nameservers explicitly, so have to rely on slirp picking a working one. If the user starts an image that has `iptables`, but is missing one of the other packages (say `sshfs`), then this allows them to install them, even if slirp DNS is broken.